middleware/locale/locale_test.go

```diff
@@ -0,0 +1,55 @@
+package locale_test
```

Comment:
@abiosoft I've already written this, but @phifty seems to be resilient when it comes to raised issues.
https://github.com/mholt/caddy/pull/781#discussion-diff-60729279